src/todo/columns.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import defaultTodoColumns from "./config.json";
+import buildColumn from "../refapp-grid/column-builder";
+
+export default function getColumns() {
+  const getDoneActionColumn = () => {
+    return [];
+  };
+
+  const getTodoActionColumn = () => {
+    const fetchEncounterURL = (patientId, encounterId) => {
+      let baseUrl: string = `${location.origin}/openmrs/htmlformentryui/htmlform/viewEncounterWithHtmlForm.page?`;
+
+      return `${baseUrl}patientId=${patientId}&encounter=${encounterId}`;
+    };
+    const getTodoActionButton = todo => {
+      switch (todo.type) {
+        case "PRINT_CONSENT":
+          return (
+            <a
+              title="Basic Service Plan Encounter URL"
```

Comment:
Can we have translated content here?

Reply:
Removed title will add a backlog card for this